documentation/Semantic-and-Model-Violations-Reference.md

```diff
@@ -845,3 +846,11 @@ This document lists the set of automated rules that can be validated against swa
 **Description**: Long running operation must return location header or azure-AsyncOperation header in response when this operation is annotated with x-ms-long-running-operation:true.
 
 **How to fix the violation**: Adding one of these headers to the response.
+
+### <a name="MISSING_RESOURCE_ID" />MISSING_RESOURCE_ID
+
+**Output Message**: id is required to return in response of GET/PUT resource calls but not provided.
```

Comment:
what ID? should it be more specific?

... is required <delete>to return</delete> in response of ....but not <add>being</add> provided